src/data/__factories__/index.js

```diff
@@ -1 +1,3 @@
-export * from './blocks';
+import getBlocksAPIResponse from './blocks';
+
+export default getBlocksAPIResponse;
```

Comment:
directly import function from the original file. No need to create an index file for just export one file.